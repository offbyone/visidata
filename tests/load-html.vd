sheet	col	row	keystrokes	input	comment
			o	sample_data/sunshinelist.html	open file
sunshinelist		0	^J		open this table
