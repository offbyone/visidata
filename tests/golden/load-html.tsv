	Year	Employer	Sector	Name	Position	Salary	Benefits	Total	Change	Change (%)
[10]	2017	Ontario Power Generation	Electricity	Jeffrey Lyash	President and Chief Executive Officer	1155899.14	9802.71	1165701.85	375741.71	47.56
[10]	2017	University of Toronto	University	William Moriarty	President and Chief Executive Officer, University of Toronto Asset Management Corporation	1045582.62	611.72	1046194.34	-429086.8	-29.09
[10]	2017	Ontario Power Generation	Electricity	Glenn Jager	Nuclear President and Chief Nuclear Officer	832750.95	4752.0	837502.95	39607.88	4.96
[10]	2017	Ontario Pension Board / Commission du Régime de retraite de l'Ontario	Crown	Jill Pepall	Executive Vice President and Chief Investment Officer / Vice-présidente directrice et chef des placements	835389.06	727.64	836116.7	94989.53	12.82
[10]	2017	University Health Network	Hospital	Peter Pisters	President and Chief Executive Officer	753992.4	77725.0	831717.4	144417.02	21.01
[10]	2017	Sunnybrook Health Sciences Centre	Hospital	Barry Mclellan	President and Chief Executive Officer	715000.01	49444.78	764444.79	-860.19	-0.11
[10]	2017	Centre for Addiction and Mental Health	Hospital	Catherine Zahn	President and Chief Executive Officer / Présidente-directrice générale	672731.19	71284.03	744015.22	None	None
[10]	2017	Baycrest Centre for Geriatric Care	Hospital	William Reichman	President and Chief Executive Officer	718475.03	14804.04	733279.07	-5285.95	-0.72
[10]	2017	St. Joseph's Healthcare Hamilton	Hospital	Kevin Smith	Chief Executive Officer, St. Joseph's Health System	660067.2	66247.71	726314.91	-22259.14	-2.97
[10]	2017	Mount Sinai Hospital	Hospital	Joseph Mapa	President and Chief Executive Officer	719724.5	1404.0	721128.5	-40428.33	-5.31
